success-criteria.md

```diff
@@ -46,8 +46,8 @@ We have a shared goal of preserving the web as an open platform for diverse and
 rich experiences provided by multiple parties. Towards this end, Improving Web
 Advertising Business Group goal is to provide monetization opportunities that
 support the open web while balancing the needs of publishers and the advertisers
-that fund them with improvements to protect people from the individual and
-societal impacts of tracking content consumption over time.
+that fund them --with improvements to protect people from the individual and
```

Comment:
This combines two separate technical concerns, contextual-based tracking activity, which does not require de-anonymizing users to perform, and behavioral-tracking which is a very different set of technical concerns, even to the extent that they are covered by regulatory policies.  . Behavioral tracking has a significantly larger scope, of which mapping consumption of content might be considered a part, but not a whole, We need to consider how technology involved in behavioral tracking represents a unique relationship to cross-site behavior and concerns. "Content consumption activity" does not define "behavior" which should also include tracking of geolocation, IP level data, system level data, Bluetooth beaconing, and network level data, all of which concern all of us in this group. I would like to discuss some changes in this phrasing to make the separation of concerns here clearer.

Reply:
> JMK: Aram correctly points out that targeting based on current content consumption is distinct from targeting based on past content consumption. However, he believes attributes based "behavior" should be expanded to the non-text-based "contextual" factors of device, connection, geolocation factors.

> I believe we should keep these important factors grouped into at least one distinct set of factors from content consumption activities--since the latter has far more privacy risks than bandwidth speed or device type choices by people. 

My response:

I firmly disagree that these factors should be grouped into a single concern. Multiple investigations have shown that bandwidth speed and device type (among a number of other behavioral identifiers, including a variety available to apps) are both primary in the use of 1 to 1 identification of devices and users and that the process of identification to a single user via their device fingerprint or geolocation has caused significantly more concern to web users than the idea that they consumption of particular content might de-anonymize them. The significant thing here is that the chance of content consumption identifying a user goes down *significantly* if that user's information is stripped of device factors. As such these are significantly different behaviors that we *must* engage on in a separate basis. 

I really don't think we can move forward unless we separate these factors (context-based tagging of users vs behavioral and fingerprint tagging) into different concerns, especially because they have radically different technical requirements.